CONTRIBUTING.md

```diff
@@ -2,59 +2,64 @@
 Welcome! Thanks for checking out Expensify.cash and for taking the time to contribute!
 
 ## Getting Started
-This guide is specifically for external contributors. For a general overview of the repo, check out our README located [here](https://github.com/Expensify/Expensify.cash/blob/master/README.md). The part of the README to pay particular attention to is how to [run the app](https://github.com/Expensify/Expensify.cash#local-development) locally using our production API.
```

Comment:
Can we add the `README.md` link back in here?

Reply:
What do you mean exactly?

Reply:
Before this change, `here` was a hyperlink to `https://github.com/Expensify/Expensify.cash/blob/master/README.md`, but after the change, it isn't a hyperlink. You can use the same markup from the old version (i.e `[here](https://github.com/Expensify/Expensify.cash/blob/master/README.md)`) to add it back in.

Reply:
@Jag96 do you just mean using the nomer `README.md` instead of `README`?

Reply:
Ignore my comment please, I commented at the same moment as you.

Reply:
Updated